connectors/vault/vault_utils/vault_utils.go

```diff
@@ -0,0 +1,213 @@
+// Copyright 2020 IBM Corp.
```

Comment:
Hard to comment since I don't know what you changed, next time please `git mv`.

Reply:
Sure thanks.  I just moved vault_utils.go to vault_utils folder and then there were some minor changes done. Will use git mv next time around.